specs/slashtags-derivation.md

```diff
@@ -0,0 +1,74 @@
+2. A public key that can be used to receive Bitcoin funds should never be the public key for a slashtag (or a public key in a keypair derived from a slashtag). 
+3. A slashtag should be generated with a semantically relevant name instead of an index number (the latter is the case with Bitcoin keypairs for creating receiving addresses). 
+
+
+## Derivation
+
+### Mnemonic words, master seed, and master node
+
+Following [BIP-39](https://bips.xyz/39), a series of mnemonic words is created (12, 15, 18, 21, or 24 words). These mnemonic words should be stored by the user of the application for the backup of their slashtags and, typically, Bitcoin funds.  
+
+Following BIP-39, this set of mnemonic words is used to produce a 512-bit master seed (also known as a "BIP-39 seed" or simply the "seed"). The master seed derivation procedure allows for increasing security via a password, which typically should be memorized or at least backed up separately from the mnemonic words.  
+
+Following [BIP-32](https://en.bitcoin.it/wiki/BIP_0032), the master seed is turned into a master node. 
+
+### From master node to PrimaryKey
+
+From the master node, a hardened child node with index "123456" is derived. The private key of this child node is referred to as the **primary key** within the Slashtags protocol (identified by the **`primaryKey`** variable within the SDK). 
+
+The index of "123456" is clearly distinct from the standard values in the purpose field which indicate various Bitcoin address formats (e.g., native segwit at "84", nested segwit at "49", and legacy at "44"). This is one way in which a slashtag derivation path is clearly distinguishable from that of a Bitcoin address.   
+
+### From primary key to slashtag seed
+
+The primary key can be used to create one or more slashtags. 
+
+To create a slashtag, the primary key is first used to produce a **slashtag seed**. It is produced by a keyed, Blake2b hash function, where the key is the primary key derived in the previous step. The output is a 32-byte Uint8Array. 
+
+The JavaScript Sodium [implementation](https://sodium-friends.github.io/docs/docs/generichashing#crypto_generichash_batch) for this keyed hash function is as follows:
+
+```js
+sodium.crypto_generichash_batch(seed, [NS, Buffer.from(name)], pk);
+```
+
+The **`seed`** variable here refers to the slashtag seed and stores the output of the function in a 32-byte Uint8Array. 
+
+The input of the function is the array. Its first element is the 32-byte buffer of the Blake2b hash value of "slashtags" (stored in the **`NS`** variable as `<Buffer 15 8a 40 6f 1f 9b e5 89 9b 0a 7f 04 71 ad eb 39 2d 3e d6 0d 24 e6 47 c5 3c c5 06 d3 f2 0f ac b4>`). The second element is the buffer of the **`name`** variable. The value of the name variable would typically have semantic meaning (e.g., "Alice", "My social media profile"). 
```

Comment:
This is less readable than the previous version with bulleted list

Reply:
Re-introduced bullet points